project_resource_calendar/models/calendar_event.py

```diff
@@ -65,6 +65,33 @@ class CalendarEvent(models.Model):
                           [('id', '!=', False), ('recurrency', '=', True), ('is_task_event', '=', True)]
                           )]}
     )
+    recurrent_state = fields.Char(
+        string='Recurring',
+        compute='_calculate_recurrent'
+    )
+    recurrence_type = fields.Char(
+        string='Recurrence Type',
+        compute='_calculate_recurrence_type'
+    )
+
+    def _calculate_recurrent(self):
+        if self.recurrency:
+            self.recurrent_state = "Yes"
+        else:
+            self.recurrent_state = "No"
+
+    def _calculate_recurrence_type(self):
+        if self.recurrency:
+            if self.end_type == 'end_date':
+                self.recurrence_type = str(self.interval) + " par " +\
+                                       str(self.rrule_type) +\
+                                       " until End Date " +\
+                                       self.final_date
+            else:
+                self.recurrence_type = str(self.interval) + " par " +\
```

Comment:
par ?  x Time(s) per week for self.count self.rrule_type --> in French (x fois par self.rrule_type pendant self.count self.rrule_type)
All strings should be translated, add the _() to be translated and Add the translation